Source/Variables/Main.lua

```diff
@@ -34,12 +34,53 @@ function Auctionator.Variables.InitializeSavedState()
   Auctionator.SavedState = AUCTIONATOR_SAVEDVARS
 end
 
+-- All "realms" that are connected together use the same AH database, this
+-- determines which database is in use.
+local function GetConnectedRealmRoot()
+  local currentRealm = GetRealmName()
+  local connections = GetAutoCompleteRealms()
+
+  -- We sort so that we always get the same first realm to use for the database
+  table.sort(connections)
+
+  -- Case where we are on a connected realm
+  if connections[1] ~= nil then
+    return connections[1]
+  -- We are not on a connected realm
```

Comment:
nitpick: can you move these comments into their respective if/else blocks? I got super confused for a minute about the logic lol because I was relating the comment to the wrong line of code.